specifications/api/swagger.yaml

```diff
@@ -1840,12 +1840,363 @@ x-responsesErr: &responsesErr
       description: A specific error code for this error, more precise than HTTP ones.
       example: an_error_code
 
+x-responsesErrWalletNotFound: &responsesErrWalletNotFound
+  type: object
+  required:
+    - message
+    - code
+  properties:
+    message:
+      type: string
+      description: A descriptive error message.
+    code:
+      type: string
+      enum: ['no_such_wallet']
+
+x-responsesErrBadRequest: &responsesErrBadRequest
+  type: object
+  required:
+    - message
+    - code
+  properties:
+    message:
+      type: string
+      description: Explanation of a good request.
```

Comment:
In a similar fashion, something that explains the root cause of the error would be better:

```
description: |
  May occur when a request is not well-formed; that is, it fails to parse
  successfully. This could be the case when some required parameters 
  are missing or, when wrong values are provided.
```